eva-accession-import-automation/validate_imported_contigs_for_assembly.py

```diff
@@ -0,0 +1,132 @@
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import click
+import hashlib
+import subprocess
+import os
+import configparser
+from __init__ import logger
+from run_accession_import import get_args_from_private_config_file
+
+
+def run_command_with_output(command_description, command):
+    process_output = ""
+
+    logger.info("Starting process: " + command_description)
+    logger.info("Running command: " + command)
+
+    with subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, bufsize=1, universal_newlines=True,
+                          shell=True) as process:
+        for line in process.stdout:
+            process_output += line
+        errors = os.linesep.join(process.stderr.readlines())
+    if process.returncode != 0:
+        logger.error(command_description + " failed!" + os.linesep + errors)
+        raise subprocess.CalledProcessError(process.returncode, process.args)
+    else:
+        logger.info(command_description + " completed successfully")
+    return process.returncode, process_output
+
+
+def get_args_from_assembly_properties_file(assembly_properties_file):
+    parser = configparser.ConfigParser()
+    parser.optionxform = str
+
+    with open(assembly_properties_file, "r") as properties_file_handle:
+        # Dummy section is needed because
+        # ConfigParser is not clever enough to read config files without section headers
+        properties_section_name = "assembly_properties"
+        properties_string = '[{0}]\n{1}'.format(properties_section_name, properties_file_handle.read())
+        parser.read_string(properties_string)
+        config = dict(parser.items(section=properties_section_name))
+        return config
+
+
+def validate_imported_contigs(assembly_properties_file, config_file):
+    config = get_args_from_private_config_file(config_file)
+    config.update(get_args_from_assembly_properties_file(assembly_properties_file))
+    config["assembly_report_path"] = config["parameters.assemblyReportUrl"].split("file:")[-1]
+    config["assembly_md5"] = hashlib.md5(config["parameters.assemblyName"].encode("utf-8")).hexdigest()
+    # We need these because string interpolation won't work if there is a dot in the key
+    config["taxonomy_accession"] = config["parameters.taxonomyAccession"]
+
+
+    config["contig_chr_mismatch_table"] = "dbsnp_ensembl_species.dbsnp_species_with_contig_chromosome_start_mismatch"
+    config["contig_chr_match_table"] = "dbsnp_ensembl_species.dbsnp_species_with_contig_chromosome_start_match"
+    final_formatting_cmd = r"cut -d$'\t' -f5 | sort | uniq | paste -s - | sed s/$'\t'/\",\"/g | sed s/$/\"/g | sed s/^/\"/g"
```

Comment:
f5 is the genbank column right? is there any way to mention that within the code without comments?